code/datums/brain_damage/split_personality.dm

```diff
@@ -238,5 +239,36 @@
 	if(objective)
 		to_chat(src, span_notice("Your master left you an objective: <b>[objective]</b>. Follow it at all costs when in control."))
 
+/datum/brain_trauma/severe/split_personality/blackout
+	name = "Blackout Drunk"
+	desc = "Patient brain is filled with alcohol and thus formed a new personality temporarily to survive."
+	scan_desc = "complete alcohol takeover"
+	gain_text = span_warning("You feel your consioucness fading...")
+	lose_text = "You have no memory of what happened while blacked out, try to trace your step and find out what happened while your were out."
+	poll_role = "black out character"
```

Comment:
```suggestion
	name = "Alcohol-Induced CNS Impairment"
	desc = "Patient's CNS has been temporarily impaired by imbibed alcohol, blocking memory formation, and causing reduced cognition and stupefaction."
	scan_desc = "alcohol-induced CNS impairment"
	gain_text = span_warning("Crap, that was one drink too many. You black out...")
	lose_text = "You wake up very, very confused and hungover. All you can remember is drinking a lot of alcohol... what happened?"
	poll_role = "blacked out drunkard"
```

Some grammar and textual nitpicks to hopefully make this more readable and reasonable.

Reply:
god damn this is good, you the man!